slurm_deploy/deploy_ctl.sh

```diff
@@ -150,7 +150,6 @@ function item_download() {
     fi
 
     config=$(echo "$config " | sed -e 's/--with-[a-z]*= //g')
```

Comment:
please remove line 152 as well, there are no reason to remove the `--with-*` arguments

Reply:
Ok